app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsFragmentTest.kt

```diff
@@ -232,145 +232,112 @@ class AdministratorControlsFragmentTest {
   }
 
   @Test
-  fun testAdministratorControlsFragment_wifiSwitchIsUncheck_autoUpdateSwitchIsUnchecked() {
+  fun testAdministratorControlsFragment_wifiSwitchIsUnchecked() {
```

Comment:
As a piece of feedback: these tests are **much** more readable--thanks for making the changes!

Reply:
thanks a lot for feedback